src/standard/builtin.ml

```diff
@@ -66,6 +66,7 @@ type _ t +=
   | Int | Integer of string
   | Rat | Rational of string
   | Real | Decimal of string
+  | Root_of_with_order of { coeffs: string list; order: string }
```

Comment:
Considering that at one point, we might need to also cover other representations, we could rather have:
```
type algebraic =
| Ordered_root of { coeffs : string list; order : string; }

type _ t +=
  | Algebraic of algebraic (* instead of Root_of_with_order *)
```

And rename all occurrences of `root_of_with_order` to something like `algebraic_ordered_root` ?